spec/models/case/base_spec.rb

```diff
@@ -478,10 +478,6 @@
     it { should_not allow_value('foobar.com').for :email  }
   end
 
-  describe '#subject' do
```

Comment:
should we be removing this? We're still performing this validation in the base model

Reply:
Good catch, thanks!